cves/kernel/CVE-2020-16166.yml

```diff
@@ -412,8 +353,9 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    The commit had multiple sign offs throughout the process.
```

Comment:
Again, this really doesn't explain how you came to the conclusion of true and is pretty vague. If possible, could you go a little more in depth so someone with lesser knowledge could understand?

Reply:
I updated the comment to be the same one I updated the other one to be. I described what a sign-off is so hopefully, that helps.